### ISSUE TITLE
ReID accuracy dropped when training with smaller batch sizes

### ISSUE THREAD

**Issue by tranhoanglam2712** [mapooon/PetFace#14]
Dear the authors of PetFace,

I have experienced a significant drop in accuracy (70% -> 28% for cats) when I was training the unified model. The only config that I changed was the batch size (256 -> 64) due to my hardware restrictions. This issue did not appear when I was training on single species with the above config. Also, I have been able to reproduce the results loading the pretrained unified models.

Do you have any suggestions on this issue?

Cheers,
Hoang Lam Tran
